src/Platforms/CUDA/CMakeLists.txt

```diff
@@ -13,9 +13,11 @@ set(CUDA_RT_SRCS CUDAfill.cpp CUDAallocator.cpp CUDAruntime.cpp)
 set(CUDA_LA_SRCS cuBLAS_missing_functions.cu)
 
 if(NOT QMC_CUDA2HIP)
-  cuda_add_library(platform_cuda_runtime ${CUDA_RT_SRCS})
-  cuda_add_library(platform_cuda_LA ${CUDA_LA_SRCS})
-  cuda_add_cublas_to_target(platform_cuda_LA)
+  add_library(platform_cuda_runtime ${CUDA_RT_SRCS})
+  add_library(platform_cuda_LA ${CUDA_LA_SRCS})
+  target_link_libraries(platform_cuda_LA PRIVATE CUDA::cublas)
+  target_include_directories(platform_cuda_runtime PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+  target_link_libraries(platform_cuda_runtime PRIVATE CUDA::cudart)
```

Comment:
Use PUBLIC and remove the above line target_include_directories

Reply:
changed